_locales/fr_fr.json

```diff
@@ -5,7 +5,7 @@
     "ABOUT/bugs/p2": " . Je pourrai ainsi le suivre.",
     "ABOUT/contributing/hdr": "contribuer ",
     "ABOUT/contributing/link": "forker DuckieTV et soumettre vos pull requests ",
-    "ABOUT/contributing/p1": "Vous souhaitez contribuer à DuckieTV et tremper dans Angular.js ? N'hésitez pas à ",
+    "ABOUT/contributing/p1": "Vous souhaitez contribuer à DuckieTV et vous impliquez dans Angular.js ? N'hésitez pas à ",
```

Comment:
Ah sorry @garfield69, here, that should be `vous impliquer` and not `vous impliquez`. I let you fix that in a commit in your side ?